Add `${siteVariables.cliVersion}` to Install the Astro CLI (#1764)
astronomer/docs: astro/cli/install-cli.md

#### Prerequisites

- [Docker Desktop](https://docs.docker.com/desktop/windows/install/) for Windows.
- [Docker Engine](https://docs.docker.com/engine/install/) (v1.13.1 or later).
- [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) enabled on your local machine.
- Astro CLI version 1.6 or later.
- The latest version of the Windows [App Installer](https://apps.microsoft.com/store/detail/app-installer/9NBLGGH4NNS1?hl=en-ca&gl=ca).
- Windows 10 1709 (build 16299) or later or Windows 11.

#### Installation

Open Windows PowerShell as an administrator and then run the following command:

```sh
winget install -e --id Astronomer.Astro
```

To install a specific version of the Astro CLI, specify the version you want to install at the end of the command. For example, running the following command specifies the latest available version of the Astro CLI:

> ```sh
> <pre><code parentName="pre">{`winget install -e --id Astronomer.Astro -v ${siteVariables.cliVersion}`}</code></pre>
> ```

Reviewer: ```suggestion
<pre><code parentName="pre">{`winget install -e --id Astronomer.Astro -v ${siteVariables.cliVersion}`}</code></pre>
```

winget install -e --id Astronomer.Astro
```

> To install a specific version of the Astro CLI, specify the version you want to install at the end of the command. For example, running the following command installs Astro CLI version ${siteVariables.cliVersion}:

Reviewer: ```suggestion
To install a specific version of the Astro CLI, specify the version you want to install at the end of the command. For example, running the following command specifies the latest available version of the Astro CLI:
```

curl -sSL install.astronomer.io | sudo bash -s
```

To install a specific version of the CLI, specify the version number as a flag at the end of the command. For example, to install the most recent release of the CLI, you would run:

> ```sh

Reviewer: ```suggestion
```

> To install a specific version of the CLI, specify the version number as a flag at the end of the command. For example, to install version ${siteVariables.cliVersion} of the CLI, you would run:

Reviewer: ```suggestion
To install a specific version of the CLI, specify the version number as a flag at the end of the command. For example, to install the the most recent release of the CLI, you would run:
```
One less item to maintain - and I'm also not sure that this will work outside of a code fence.

> ```sh

Author: @jake - this change has been implemented.

<pre><code parentName="pre">{`curl -sSL install.astronomer.io | sudo bash -s -- v${siteVariables.cliVersion}`}</code></pre>

> ```

Reviewer: ```suggestion
```

</Tabs>

> ### Step 2: Install Astro CLI version ${siteVariables.cliVersion}

Reviewer: ```suggestion
### Step 2: Install the Astro CLI 
```
